src/components/PreviewDraft/PreviewDraftLightboxV2.tsx

```diff
@@ -45,6 +45,7 @@ interface MarkupPreviewProps extends BaseProps {
 interface VersionPreviewProps extends BaseProps {
   type: 'version';
   article: IArticle;
+  customTitle?: string;
```

Comment:
Hvor mange custom titler forventes det at denne mottar? Vil tittelen alltid styres av status; at den er published eller en versjon? Hvis sistnevnte; er det bedre at dette flagget blir en boolean, med navnet isPublished, og at h2-elementet bare switcher mellom den ferdige published-tittelen og version-tittelen?

Reply:
Ja, det er også en måte å gjøre det på! Ikke superfan av noen av måtene å gjøre det på egentlig, meen det gjør at vi får gjenbrukt komponenten i alle fall .. På nåværende tidspunkt så vil det kun være to ulike titler som vises, men tenkte det likevel var en god ide å gjøre det mest mulig generisk! 

Syntes `isPublished` er et litt misvisende navn isåfall, fordi begge variantene sammenligner mot en publisert versjon. Kunne nok kommet opp med et annet navn, men vet ikke om det gjør koden så mye mer forståelig!